### DISCUSSION TITLE
Does Github copilot have limited code lines?

### DISCUSSION THREAD

**Discussion by N4SIRODDIN3** [community/community#14057] [Copilot Conversations]
Hello, Pilots.
It seems that GitHub copilot suggestions get truncated when the code exceeds specific lines.
Does Github copilot have limited code lines on its proposals?

- **Steps to reproduce:**

Open a new file with VScode with the Copilot extension enabled, choose a language (python my case), write the following code:
```py
# optimize the beam depth using the particle swarm algorithm and OpenSEES
class bea
```

- **Expected behavior:**

A complete algorithm inside the class.

- **I'm using:**

Visual Studio Code (64-bit), Version 1.64
Copilot version v1.10.5413

**Comment by N4SIRODDIN3** [comment]
@github-staff

**Comment by CaptainPhilipp** [comment]
same problem. the feature becomes almost unusable with limited output
